content/it/_index.md

```diff
@@ -0,0 +1,25 @@
+
+---
+title: "Glossario Cloud Native"
+---
+
+# Glossario Cloud Native
+
+Il Glossario Cloud Native è un progetto guidato dal Business Value Subcommittee (BVS) della CNCF con cui la community si propone di spiegare concetti del mondo cloud native con un linguaggio semplice e diretto che non presupponga conoscenze tecniche pregresse. La [versione in PDF può essere visualizzata e scaricata qui] (https://github.com/cncf/glossary/blob/main/cloudnative-glossary.pdf).
```

Comment:
We removed the last sentence with the reference to the PDF as it's outdated.

Reply:
done!